future-apps/payroll/contracts/Payroll.sol

```diff
@@ -443,24 +596,62 @@ contract Payroll is AragonApp { //, IForwarder { // makes coverage crash (remove
                 1,
                 ""
             );
-            SendPayroll(employee.accountAddress, token, tokenAmount);
+            emit SendPayroll(employee.accountAddress, token, tokenAmount);
             somethingPaid = true;
         }
+
+        // try to remove employee
```

Comment:
@bingen was there a reason to remove terminated employees, rather than leaving them? Gas refund?

Reply:
Hmm... re-enabling could be interesting. The only complication I could see with it is dealing with accrued balances during re-enabling.